.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This is invalid.

Reply:
This is a default template. What's wrong with it?

Reply:
Comments aren't allowed in JSON.

Reply:
I see, fixed in the main branch.